### ISSUE TITLE
Javascript für yform Link-Widget prüfen

### ISSUE THREAD

**Issue by joachimdoerr** [FriendsOfREDAXO/mform#321]
Nach einem fetten merge von develop nach master gab es hunderte changes und nochmal so viele mergekonflikte, ich hab getan was ich konnte um das javascript so plausibel wie möglich zu mergen. es wäre aber sicher sehr gut wenn @ynamite wenn du das nochmal prüfen würdest.

**Comment by ynamite** [comment]
Das sieht auf den ersten Blick und kurzem Vergleich mit der vorherigen Version korrekt aus.
Ich werde es heute oder morgen noch testen und melde mich nochmal.

**Comment by joachimdoerr** [comment]
@ynamite passt das wie es ist? Konntest Du es auch testen?

**Comment by ynamite** [comment]
@joachimdoerr yep, habe es eben mit der Version 7.4.4 getestet. Die Auswahl eines yform Datensatzes klappt aus mform und yform aus. Allerdings werden vor dem custom_link widget in yform folgende Warnungen ausgegeben:
```
Warning: Undefined array key 1 in src/addons/mform/lib/Widget/var_custom_link.php on line 112
Warning: Undefined array key 2 in src/addons/mform/lib/Widget/var_custom_link.php on line 113
```
Die Definition für das yform link widget ist wie folgt
`test::rex_yf_portfolio_tag::name,Name`

**Comment by joachimdoerr** [comment]
@ynamite bitte teste doch mit der aktuellen MForm 8.0.0-Beta6.

**Comment by ynamite** [comment]
Mit der 8.0.0-Beta6 habe ich folgende Warnungen.

Modul-Input custom_link:
`Warning: Array to string conversion in src/addons/mform/lib/MForm/Parser/MFormParser.php on line 921`

Yform custom_link:
```
Warning: Undefined array key 1 in src/addons/mform/lib/Widget/var_custom_link.php on line 108
Warning: Undefined array key 2 in src/addons/mform/lib/Widget/var_custom_link.php on line 109
```

Aber auch hiermit klappt bei mir alles

**Comment by joachimdoerr** [comment]
ok perfekt danke @ynamite die warnings werd ich mir nochmal anschauen.

**Comment by joachimdoerr** [comment]
@ynamite ich hab mir das mal angeschaut und das yform widget wird ganz sicher falsch angesprochen siehe screenshot.
<img width="1312" alt="Bildschirmfoto 2024-05-13 um 23 26 35" src="https://github.com/FriendsOfREDAXO/mform/assets/1191549/e70fdd53-3026-487d-be91-07f0583437c7">

**Comment by ynamite** [comment]
Hm, bei mir klappt es nach wie vor. Bei mir sieht die URL aber so aus:
`index.php?page=yform/manager/data_edit&table_name=rex_yf_portfolio_tag&rex_yform_manager_opener[id]=cl96b86ce4f54d&rex_yform_manager_opener[field]=name&rex_yform_manager_opener[multiple]=0`

**Comment by joachimdoerr** [comment]
@ynamite dank dir, bei mir passt jetzt auch alles es gab noch eine Kleinigkeit die jetzt erledigt ist: 6ffc9b084213d9c5cc5ee817a375f2d06f0b9f2a
